### PR TITLE
Add Tarun's NewDelhi File

### LINKED ISSUE #7: tarungupta70 tracks progress on GH Flow
- [x] Create a branch
- [x] Add File/creating commits
- [x] Open a PR
- [x] Have a discussion about PR
- [x] Add commits
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by tarungupta70**
@hectorsector 
1. Are there any other places you think I should mention?

closes #7

### CODE REVIEW

File: tarungupta70-newdelhi.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## New Delhi 
+
+### Places to Eat
+- Karim's (Old Delhi)
+
+### Places to Go
+- Red Fort
+
+### Fun Factors
```
Comment (githubteacher) [inline tarungupta70-newdelhi.md:9]: What is the population of New Delhi?

**Comment by humbertsuarez** [comment]
How hot is New Delhi?

File: tarungupta70-newdelhi.md (hunk 1 above)
Comment (soominheo3268) [inline tarungupta70-newdelhi.md:9]: https://en.wikipedia.org/wiki/New_Delhi

File: tarungupta70-newdelhi.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## New Delhi 
+
+### Places to Eat
```
Comment (humbertsuarez) [inline tarungupta70-newdelhi.md:3]: Leave a comment

Comment (humbertsuarez) [inline tarungupta70-newdelhi.md:3]: This is a line note

**Pull request merged**